manim/utils/tex_file_writing.py

```diff
@@ -304,7 +306,9 @@ def print_all_tex_errors(log_file: Path, tex_compiler: str, tex_file: Path) -> N
 ]
 
 
-def print_tex_error(tex_compilation_log, error_start_index, tex_source):
+def print_tex_error(
+    tex_compilation_log: list[str], error_start_index: int, tex_source: list[str]
+) -> None:
```

Comment:
According to the way the parameters are used inside the function, they could be `Sequence`s:
```suggestion
def print_tex_error(
    tex_compilation_log: Sequence[str],
    error_start_index: int,
    tex_source: Sequence[str],
) -> None:
```